great_expectations/rule_based_profiler/types/builder.py

```diff
@@ -0,0 +1,73 @@
+import json
+from typing import Any
+
+import great_expectations.exceptions as ge_exceptions
+from great_expectations.core.util import convert_to_json_serializable
+from great_expectations.types import SerializableDictDot
+from great_expectations.util import deep_filter_properties_iterable
+
+
+class Builder(SerializableDictDot):
```

Comment:
This is to be removed right? Can we make a note about that?

Reply:
@cdkini I made a note to myself -- this will be addressed soon.